packages/design-system/src/styles/components/_Table.scss

```diff
@@ -80,8 +40,9 @@
       // stylelint-enable selector-no-qualifying-type
       flex: none;
       font-weight: bold;
-      max-width: 120px;
-      width: 50%;
+      max-width: $table-stacked-header-max-width;
+      width: 80%;
+      word-wrap: break-word;
     }
 
     > * {
```

Comment:
We can handle this in a followup PR, but in general this scss file is pretty hard to read. I think it would be really helpful to add more comments to label each selector, and describe some of the code here